metricbeat/module/haproxy/info/_meta/fields.yml

```diff
@@ -0,0 +1,196 @@
+- name: info
+  type: group
+  description: >
+    General infomration collected on HAProxy process
+  fields:
+    - name: nb_proc
+      type: integer
+      description: >
+        Number of processes
+
+    - name: process_num
+      type: integer
+      description: >
+        Process number
+
+    - name: pid
+      type: integer
+      description: >
+        Process ID
+
+    - name: uptime_sec
+      type: integer
+      description: >
+        Current uptime in seconds
+
+    - name: mem_max_mb
+      type: integer
+      format: bytes
```

Comment:
If the value is actually in MB then you don't want to format it as `bytes`. I recommend converting the value to bytes and renaming the field so you can take advantage of the byte formatter in Kibana (it uses http://numeraljs.com/).